src/frontend/hooks/use-record/params-to-form-data.ts

```diff
@@ -37,6 +37,11 @@ function paramsToFormData(params: Record<string, any>): FormData {
       }
       return formData.set(key, FORM_VALUE_EMPTY_OBJECT)
     }
+    // Convert Date fields to UTC timezone
+    if (Date.parse(value)) {
```

Comment:
I think we need a different check for Date here because for example `Date.parse(11)` is truthy while it should only convert valid date strings

Reply:
Yes, you are right. Let me look for other way.

Reply:
Hi @dziraf, sorry for the late answer. I haven't could find an optimal solution. The solution that I found, without using an external lib, is to use regex to find the date string that returns the date picker. So, as the date picker returns a string with the format `YYYY-MM-DD HH:mm` we could use the conditional:
```if (typeof value === 'string' && value.search(/^([0-9]{4}-[0-9]{2}-[0-9]{2}\s[0-9]{2}:[0-9]{2})$/g) !== -1)```

Other solution is to use moment library with the method `isValid`, for example:
```if (moment(value, 'YYYY-MM-DD HH:mm', true).isValid())```

These solutions, of course, work because I am assuming that the date picker always will return the date in the format `YYYY-MM-DD HH:mm`, so if the format changes, the solutions won't work. That is why I don't like them very much.  

Let me know if you like one of the options or if you found a better solution, so I can update the PR.

Reply:
I was viewing the DatePicker component. And actually, the date string always will be in the format `YYYY-MM-DD HH:mm` because admin-bro is formatting the value in the onChange callback:

```
const onDateChange = (newDate: Date): void => {
  if (!disabled) {
    onChange(formatDateProperty(newDate, propertyType))
  }
}
```
https://github.com/SoftwareBrothers/admin-bro-design-system/blob/137b295e0db7ac5bc2827ca897e1e6d546df8e53/src/molecules/date-picker/useDatePicker.ts#L34-38

```
const formatDate = (date: Date): string =&gt; `${date.getFullYear()}-${pad(date.getMonth() + 1)
const formatTime = (date: Date): string => `${pad(date.getHours())}:${pad(date.getMinutes())}`
const formatDateTime = (date: Date): string => `${formatDate(date)} ${formatTime(date)}`
```
https://github.com/SoftwareBrothers/admin-bro-design-system/blob/master/src/utils/date-utils.ts

Reply:
I think what @dziraf meant by this, is that a user could feasibly write something date-ish ("YYYY-MM-DD HH:MM") that would be caught by that `if` and turned into an ISO string.

This isn't fixed, right? It seems like if someone types a string that passes this regex, it gets turned into an ISO string.

Instead of checking the string format, perhaps it should only accept `Date` objects here?

Reply:
Left a suggestion to illustrate

Reply:
Yes, the problem is that the DatePicker returns a string instead of a `Date` object. Maybe the correct way should be to modify the DatePicker component to returns a Date object.

Reply:
Probably, yes

Reply:
I did the changes, now remains to modify the DatePicker output.